src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -0,0 +1,57 @@
+package nextstep.subway.line.domain;
+
+import javax.persistence.Embeddable;
+import java.util.Objects;
+
+@Embeddable
+public class Distance {
+    private static final int MIN_VALUE = 1;
+
+    private int distance;
+
+    protected Distance() {
```

Comment:
```suggestion
    protected Distance() {}
```
중요한 부분은 아니지만 [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 작성해줄 수도 있을 것 같아요..!